internal/storetest/storedata.go

```diff
@@ -115,6 +116,24 @@ func (storeData *StoreData) LoadTuples(basePath string) error {
 		}
 	}
 
+	if storeData.TupleFiles != nil {
```

Comment:
@rhamzeh Can you take a look at this if possible?

This would be a backwards compatible solution without messing with the way the Tuples are read for a single tuple file

Reply:
Currently this would cause an issue if both the tuple file and tuple files array are being passed

Reply:
We can choose to directly have the files in the list format inside tupleFiles
and remove separate file/files altogether